Makefile

```diff
@@ -36,8 +36,21 @@ apply-with-kind: verify build-with-kind ## Deploy the kwok controller from the c
 		$(HELM_OPTS) \
 		--set controller.image.repository=$(IMG_REPOSITORY) \
 		--set controller.image.tag=$(IMG_TAG) \
+		--set serviceMonitor.enabled=true \
 		--set-string controller.env[0].name=ENABLE_PROFILING \
-		--set-string controller.env[0].value=true
+		--set-string controller.env[0].value=true 
+
+e2etests: ## Run the e2e suite against your local cluster
+	cd test && go test \
+		-p 1 \
+		-count 1 \
+		-timeout 3h \
```

Comment:
Do we still need this long of a timeout?

Reply:
Good call. Just copied from AWS provider.